components/Blog/blog.scss

```diff
@@ -0,0 +1,61 @@
+$side-nav-bg-color: #f9f9f9;
```

Comment:
This file should be a partial in our assets file, like the rest of the scss files. The stylesheets should be able to co-exist and share variables to make this more maintainable.